### PR TITLE
Javascript/javascript3/week1

### PR DESCRIPTION
**Pull request by Gohar33**

### CODE REVIEW

File: javascript/javascript3/week1/giphy.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const input1 = document.querySelector(".search")
```
Comment (aina21) [inline javascript/javascript3/week1/giphy.js:1]: Try to use meaningful names for variables.

File: javascript/javascript3/week1/giphy.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const button = document.querySelector("button")
+const gifCount = document.querySelector(".gif-number")
+document.addEventListener("DomContentLoaded", getGif)
+
+function getGif() {
+    button.addEventListener("click", function (content) {
+        content.preventDefault();
+        let inputVal = input1.value.trim()
```
Comment (aina21) [inline javascript/javascript3/week1/giphy.js:9]: const

File: javascript/javascript3/week1/giphy.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        let url = `https://api.giphy.com/v1/gifs/search?api_key=TJRmCZYdbW1KDE0BZUYGwuRwksC8RjFu&q=${inputVal}&limit=${gifCount.value}&offset=0&rating=g&lang=en`
+        url = url.concat(inputVal)
```
Comment (aina21) [inline javascript/javascript3/week1/giphy.js:11]: why do you need to concat?

File: javascript/javascript3/week1/giphy.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+        fetch(url)
+            .then(response => response.json())
+            .then(gif => {
+                let myDiv = document.getElementById("my-div")
```
Comment (aina21) [inline javascript/javascript3/week1/giphy.js:16]: const

File: javascript/javascript3/week1/giphy.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+                myDiv.innerHTML = ""
+
+                gif.data.forEach(giphy => {
+                    let imgGif = document.createElement("img")
```
Comment (aina21) [inline javascript/javascript3/week1/giphy.js:20]: const

Comment (aina21) [reply javascript/javascript3/week1/giphy.js:20]: Always use const until it won't let you :slightly_smiling_face:

File: javascript/javascript3/week1/homework.basic.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+// Create your own json file
+
+const myFavoriteMusic = '{"name":"Favorite Music", "genres": [ "electronic", {"rock":["Russian", "Classic"]}, "classical"], "bands": ["DDD", "Pink Floyd", "Metallica"], "performers":["Zemfira", "Nils Frahm", "Christian Löffler", "John Hopkins"]}';
```
Comment (aina21) [inline javascript/javascript3/week1/homework.basic.js:3]: this object should be in json file

File: javascript/javascript3/week1/weather.js [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+//Weather app
+
+const input = document.getElementById("hey")
+const button = document.querySelector("button")
+const weatherResult = document.querySelector(".weather-container")
+
+function getCityWeather() {
+    button.addEventListener("click", function () {
+        let inputVal = input.value
```
Comment (aina21) [inline javascript/javascript3/week1/weather.js:11]: const, and it's better to change a name to something meaningful.

File: javascript/javascript3/week1/weather.js [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        let url = `http://api.openweathermap.org/data/2.5/weather?q=${inputVal}&units=metric&apikey=0284346df0317b81c2e17a35dfbda0c4`
```
Comment (aina21) [inline javascript/javascript3/week1/weather.js:12]: same

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :)

File: javascript/javascript3/week1/weather.js (hunk 8 above)
Comment (Gohar33) [reply javascript/javascript3/week1/weather.js:12]: Thank you Ida for your feedback, I'll implement this.

**Pull request merged**